### DISCUSSION TITLE
Setting for the offset Value

### DISCUSSION THREAD

**Discussion by cloudskylimit** [aahnik/tgcf#45] [Q&A] [answered]
Hello, It's me again.
First thanks for the response of my QnA before.
For now, if I set offset value = 0, then it will start forward from the beginning of the chat/group chat.
I would like to know if, can I start forwarding from the latest message?
🙏

**Comment by aahnik** [comment, marked as answer]
What do you want?

1. Start forwarding from the latest message, and go backward, forwarding all the past messages.
or
2. Wait for any new incoming message and when it arrives, forward it.

Whatever you desire, please read this #43 

------

Update:

Read about two modes that tgcf now supports: [Live and past](https://github.com/aahnik/tgcf/wiki/Past-vs-Live-modes-explained)
Also read how to [deploy to cloud](https://github.com/aahnik/tgcf#run-on-cloud-%EF%B8%8F)

**Reply by cloudskylimit** [reply]
I think the 2nd one. is it same with the Live Syncing that you mention at #43 ?

**Reply by aahnik** [reply]
yes, that is the live-syncing.

**Reply by aahnik** [reply]
Live syncing is free now. Read the readme
